server/controllers/temporary-accommodation/manage/bedspaceSearchController.ts

```diff
@@ -40,11 +41,14 @@ export default class BedspaceSearchController {
             startDate,
             durationDays,
           } as BedSearchParameters)
+
+          tableRows = BedspaceResultTablePresenter.present(results.results)
```

Comment:
I feel like I'd like to discuss before taking this approach - feels like it's introducing a third way of doing things

- Doing formatting within a service, called from the controller
- Exposing functions like thing to Nunjucks and calling them directly within the template (in CAS3 we've done this for self-contained components, CAS1 has moved towards this a bit more extensively - see `nunjucksSetup.ts` in each)
- Doing formatting within a presenter, called from the controller

My feeling is I'd rather we do option 2 here (and it feels tidier to me to only have the one thing we're passing in that may-or-may-not be null depending on whether a search has been entered), but lmk what you think

With option 2, we'd have something like:

```
{{ govukTable({
  head: tableHeadings,
  rows:  BedspaceSearchResultUtils.resultTableRows(results)
}) }}
```
with the controller staying unchanged